modules/services/cloud-connector-ecs/variables.tf

```diff
@@ -54,8 +56,9 @@ variable "existing_cloudtrail_config" {
     If there's an existing cloudtrail, input mandatory attributes, and one of the 1 or 2 labeled optionals.
     <ul>
       <li>cloudtrail_sns_arn: Optional 1. ARN of a cloudtrail-sns topic. If specified, deployment region must match Cloudtrail S3 bucket region</li>
-      <li>cloudtrail_s3_sns_sqs_arn: Optional 2. ARN of the queue that will ingest events forwarded from an existing cloudtrail_s3_sns</li>
-      <li>cloudtrail_s3_sns_sqs_url: Optional 2. URL of the queue that will ingest events forwarded from an existing cloudtrail_s3_sns<br/>sqs:ReceiveMessage and sqs:DeleteMessage permissions have to be provided to the compute role</li>
```

Comment:
this is bit messy, our fault :)

The idea with the "Optional X" is to group disjointed options. Currently there are two options (cloudtrail_sns OR cloudtrail_s3).

Because the `cloudtrail_kms_arn`  configuration applies to both, I would add it in the beginning.
Let's see if changing 1/2 for CloudltrailSns/CloudtrailS3 makes it more clear.
```
<ul>
      <li>cloudtrail_kms_arn: Optional. ARN of a cloudtrail KMS key used for encrypting the logs. Required in order to retrieve the encrypted logs from S3</li>
      <li>cloudtrail_sns_arn: Optional CloudTrailSns. ARN of a cloudtrail-sns topic. If specified, deployment region must match Cloudtrail S3 bucket region</li>
      <li>cloudtrail_s3_sns_sqs_arn: Optional CloudtrailS3. ARN of the queue that will ingest events forwarded from an existing cloudtrail_s3_sns</li>
      <li>cloudtrail_s3_sns_sqs_url: Optional ClouldtrailS3. URL of the queue that will ingest events forwarded from an existing cloudtrail_s3_sns<br/>sqs:ReceiveMessage and sqs:DeleteMessage permissions have to be provided to the compute role</li>
</ul>
```